posthog/temporal/batch_exports/squash_person_overrides.py

```diff
@@ -437,17 +470,14 @@ async def squash_events_partition(inputs: QueryInputs) -> None:
 
     query = SQUASH_EVENTS_QUERY
 
-    latest_created_at = inputs.latest_created_at.timestamp() if inputs.latest_created_at else inputs.latest_created_at
+    latest_created_at = inputs.latest_created_at
 
     for partition_id in inputs.partition_ids:
         activity.logger.info("Executing squash query on partition %s", partition_id)
 
         parameters = {
             "partition_id": partition_id,
             "team_ids": inputs.team_ids,
-            # We pass this as a timestamp ourselves as clickhouse-driver will drop any microseconds from the datetime.
-            # This would cause the latest merge event to be ignored.
-            # See: https://github.com/mymarilyn/clickhouse-driver/issues/306
             "latest_created_at": latest_created_at,
         }
```

Comment:
It's not clear to me why the way `QueryInputs.last_created_at` is treated here changed, while this other usage didn't: https://github.com/PostHog/posthog/blob/270bff77be34dafc686055d484119d1df8e897b5/posthog/temporal/batch_exports/squash_person_overrides.py#L372-L381

Reply:
New client handles timestamps without losing precision, so we don't need to `timestamp()` it ourselves. Nothing is changing, mostly just pushing the operation until the last moment.